oktapam/constants/descriptions/links.go

```diff
@@ -5,6 +5,7 @@ const (
 	LinkADConnection          = "For more information check out the [documentation](https://help.okta.com/asa/en-us/Content/Topics/Adv_Server_Access/docs/ad-connections.htm) on AD Connection creation and usage."
 	LinkADRuleAssignment      = "For more information check out the [documentation](https://help.okta.com/asa/en-us/Content/Topics/Adv_Server_Access/docs/ad-sync-assignment.htm) on AD rule assignment."
 	LinkADServerDiscovery     = "For more information check out the [documentation](https://help.okta.com/asa/en-us/Content/Topics/Adv_Server_Access/docs/ad-sync.htm) on AD server discovery."
+	LinkADUserDiscovery       = "For more information check out the [documentation](https://help.okta.com/asa/en-us/Content/Topics/Adv_Server_Access/docs/ad-sync.htm) on AD user discovery."
 	LinkGateway               = "For details, see [Advanced Access Server Gateways](https://help.okta.com/asa/en-us/Content/Topics/Adv_Server_Access/docs/gateways-and-bastions.htm) on ASA Gateways and bastions."
```

Comment:
Will need followup.

Reply:
Talk to Walter on this, we can have something similar to Kubernetes Cluster and mark this as Beta Feature.

Reply:
We can update this link later once we have documentation available.

Reply:
opened https://oktainc.atlassian.net/browse/OKTA-593282